assets/data/community.json

```diff
@@ -364,5 +364,12 @@
         "course_stage": "Alumni",
         "favorite_language": "Python",
         "currently_learning": "React Native"
-    }
+    },
+    	{
+		"name": "Brian",
```

Comment:
Hey @BrianWhelanDublin this is a great PR 🙌 I do have considerations for the indentation though across the files.

For example, in the `community.json` file, your own JSON objects inclusion is indented by almost double the normal? Can this be brought in line with the rest of the indentation for the file? 😄 🤜